eos-tech-support/eos-download-image

```diff
@@ -258,7 +260,8 @@ class EosDownloadImage(object):
                        action='store_true',
                        help='Fetch images from the Endless internal network')
         p.add_argument('-s', '--size', type=int,
-                       help='Size in bytes of compressed image (use with --url)')
+                       help='Expected size in bytes of compressed image (for '
+                            'use with --url)')
```

Comment:
I kinda feel like all of this could go away since the size from the HEAD request is preferred, anyways. But I guess that makes it a bigger change.